pennylane/ops/qubit/state_preparation.py

```diff
@@ -151,7 +151,7 @@ class QubitStateVector(StatePrep):
     ndim_params = (1,)
     """int: Number of dimensions per trainable parameter of the operator."""
 
-    def __init__(self, state, wires, do_queue=True, id=None):
+    def __init__(self, state, wires, do_queue=None, id=None):
```

Comment:
Could add `do_queue` and `id` to the "Args" section above.

Reply:
94915cd8a1748832d3d6d4ef3b35423e7b8ecb0b